Update documentation: accouns (#1383)
rug-cit-hpc/league-of-robots: roles/online_docs/templates/mkdocs/docs/request-an-account.md

#jinja2: trim_blocks:False
# Request an account via mail

## Instructions

**Copy the template mail below and fill all the fields** (the yes/no field should have yes OR no).
If everything is filled, please send it back to the HPC helpdesk and request access of the group owners (preferably in the same email by cc'ing (all) the group owner(s).

**NOTE**

 - Make sure you include all group owners in the CC of the request - it will be approved only after the group owners approve access to their own group(s)

>  - All of the fields are mandatory for the account creation, failing to provide any of them will result in account not being created<br>

Reviewer: ```suggestion
 - All of the fields are mandatory for the account creation, failing to provide any of them will result in account not being created
```

#jinja2: trim_blocks:False
# Request an account via mail

## Instructions

**Copy the template mail below and fill all the fields** (the yes/no field should have yes OR no).

> If everything is filled, please send it back to the HPC helpdesk and request access of the group owners (preferably in the same email by cc'ing (all) the group owner(s).<br>

Reviewer: ```suggestion
If everything is filled, please send it back to the HPC helpdesk and request access of the group owners (preferably in the same email by cc'ing (all) the group owner(s).
```

**NOTE**

 - Make sure you include all group owners in the CC of the request - it will be approved only after the group owners approve access to their own group(s)<br>
 - All of the fields are mandatory for the account creation, failing to provide any of them will result in account not being created<br>
 - End date is valid if it is requested/confirmed by group owner (or department's secretary by sending email directly to the hpc.helpdesk@umcg.nl<br>

>  - Code Of Conduct (COC) is available at the [https://docs.gcc.rug.nl/nibbler/coc_umcg_research_clusters/](https://docs.gcc.rug.nl/nibbler/coc_umcg_research_clusters/)<br>

Reviewer: ```suggestion
 - Make sure you include all group owners in the CC of the request - it will be approved only after the group owners approve access to their own group(s).
 - All of the fields are mandatory for the account creation, failing to provide any of them will result in account not being created.
 - End date is valid if it is requested/confirmed by group owner (or department's secretary by sending email directly to the hpc.helpdesk@umcg.nl.
 - Code Of Conduct (COC) is available at the [https://docs.gcc.rug.nl/nibbler/coc_umcg_research_clusters/](https://docs.gcc.rug.nl/nibbler/coc_umcg_research_clusters/).
```

Reviewer: Note: do not mix HTML tags in MarkDown templates that get converted into HTML as that can produce unexpected results. If you need a line break inside a paragraph or list item, then use the corresponding MarkDown syntax. That is not the case here though.

## Template email (copy everything)

```
Dear groupowner and helpdesk,

@the groupowners can you approve my access to your group by replying to this email.

I would like to request an access account to the cluster.
 - First Name: 
 - Last Name:  
 - Email address: 
 - Public key: 
 - End date contract (YYYY-MM-DD):
 - Groups to access:
   - 
 - I have read, understood and I agree to the UMCG HPC Code of Conduct:	yes/no

Best regards,
[NAME]
```

> and send to helpdesk and group owner(s)

Reviewer: ```suggestion
```
